[2025-02-19T14:22:49.762-0600] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-19T14:22:49.788-0600] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: DAG_1.send_message scheduled__2025-02-19T05:08:00+00:00 [queued]>
[2025-02-19T14:22:49.801-0600] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: DAG_1.send_message scheduled__2025-02-19T05:08:00+00:00 [queued]>
[2025-02-19T14:22:49.802-0600] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-02-19T14:22:49.839-0600] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): send_message> on 2025-02-19 05:08:00+00:00
[2025-02-19T14:22:49.848-0600] {standard_task_runner.py:72} INFO - Started process 495634 to run task
[2025-02-19T14:22:49.855-0600] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'DAG_1', 'send_message', 'scheduled__2025-02-19T05:08:00+00:00', '--job-id', '403', '--raw', '--subdir', 'DAGS_FOLDER/dag_1.py', '--cfg-path', '/tmp/tmpmraa43je']
[2025-02-19T14:22:49.861-0600] {standard_task_runner.py:105} INFO - Job 403: Subtask send_message
[2025-02-19T14:22:49.948-0600] {task_command.py:467} INFO - Running <TaskInstance: DAG_1.send_message scheduled__2025-02-19T05:08:00+00:00 [running]> on host IMNOTROY.
[2025-02-19T14:22:50.121-0600] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='DAG_1' AIRFLOW_CTX_TASK_ID='send_message' AIRFLOW_CTX_EXECUTION_DATE='2025-02-19T05:08:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-19T05:08:00+00:00'
[2025-02-19T14:22:50.123-0600] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-19T14:22:50.153-0600] {python.py:240} INFO - Done. Returned value was: Done!
[2025-02-19T14:22:50.194-0600] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-19T14:22:50.195-0600] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=DAG_1, task_id=send_message, run_id=scheduled__2025-02-19T05:08:00+00:00, execution_date=20250219T050800, start_date=20250219T202249, end_date=20250219T202250
[2025-02-19T14:22:50.315-0600] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-19T14:22:50.346-0600] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-19T14:22:50.348-0600] {local_task_job_runner.py:245} INFO - ::endgroup::
